.circleci/config.yml

```diff
@@ -6,17 +6,69 @@ jobs:
 
   build-python36:
     docker:
-      - image: ocrd/core
+      - image: ubuntu:18.04
     steps:
-      - run: apt-get update && apt-get install -y --no-install-recommends make git curl
+      - run: apt-get update
+      - run: apt-get install -y software-properties-common
+      - run: add-apt-repository -y ppa:deadsnakes/ppa
+      - run: apt-get update && apt-get install -y --no-install-recommends make git curl python3.6
       - checkout
       - run: make deps-ubuntu
-      - run: make install
-      - run: make test-cli
-      - run: make coverage
+      - run: make install PYTHON=python3.6
+      - run: make test-cli PYTHON=python3.6
+      - run: make coverage PYTHON=python3.6
+      - codecov/upload
+
+  build-python37:
+    docker:
+      - image: ubuntu:18.04
+    steps:
+      - run: apt-get update
+      - run: apt-get install -y software-properties-common
+      - run: add-apt-repository -y ppa:deadsnakes/ppa
+      - run: apt-get update && apt-get install -y --no-install-recommends make git curl python3.7
+      - checkout
+      - run: make deps-ubuntu
+      - run: make install PYTHON=python3.7
+      - run: make test-cli PYTHON=python3.7
+      - run: make coverage PYTHON=python3.7
+      - codecov/upload
+
+
+  build-python38:
+    docker:
+      - image: ubuntu:18.04
+    steps:
+      - run: apt-get update
+      - run: apt-get install -y software-properties-common
+      - run: add-apt-repository -y ppa:deadsnakes/ppa
+      - run: apt-get update && apt-get install -y --no-install-recommends make git curl python3.8
+      - checkout
+      - run: make deps-ubuntu
+      - run: make install PYTHON=python3.8
+      - run: make test-cli PYTHON=python3.8
+      - run: make coverage PYTHON=python3.8
+      - codecov/upload
+
+  build-python39:
+    docker:
+      - image: ubuntu:18.04
+    steps:
+      - run: apt-get update
+      - run: apt-get install -y software-properties-common
+      - run: add-apt-repository -y ppa:deadsnakes/ppa
+      - run: apt-get update && DEBIAN_FRONTEND=noninteractive TZ=Europe/Berlin apt-get install -y --no-install-recommends make git curl python3.9
```

Comment:
Why `TZ=Europe/Berlin`, why only here?

Reply:
Because https://docs.python.org/3/library/zoneinfo.html was added which makes the `python3.9` package install tzinfo which causes a prompt to choose a timezone.

Reply:
Is now set for all versions.